config/dr-cluster/manifests/odr/kustomization.yaml

```diff
@@ -6,6 +6,9 @@ resources:
 
 patches:
 - patch: |-
+    - op: add
+      path: /metadata/annotations/operatorframework.io~1suggested-namespace
+      value: openshift-dr-cluster
     - op: replace
       path: /metadata/name
       value: odr-cluster.v0.0.0
```

Comment:
I think this needs to change as well to `odr-cluster-operator` @raghavendra-talur

Reply:
I agree, this needs to change too.